plugins/usbdmx/ShowJockeyDMXU1.cpp

```diff
@@ -83,7 +83,7 @@ bool LocateInterface(LibUsbAdaptor *adaptor,
         bool isOutput = (current_endpoint_addr & LIBUSB_ENDPOINT_DIR_MASK) !=
                         LIBUSB_ENDPOINT_IN;
         if (endpoint->bmAttributes == LIBUSB_TRANSFER_TYPE_BULK && isOutput) {
-          OLA_INFO << "Using interface " << i;
+          OLA_INFO << "Using interface " << i <<"at address "<<current_endpoint_addr;
```

Comment:
Can you add spaces around the chevrons please. You also need a space before "at" for it to print out as intended, otherwise it will come out as iat.